src/pages/DetailsPage/DetailsPage.tsx

```diff
@@ -26,9 +28,16 @@ const DetailsPage: FunctionComponent = () => {
   const user: User = queryClient.getQueryData('user');
   const { isLoading, error, data } = useSingleProduct(SKU);
   const tableIsEnabled = useFeatureFlag('subscriptionInventory.detailsTable');
-
   const missingText = 'Not Available';
 
+  const handle404Error = (error: unknown) => {
+    if (String(error).includes('404')) {
+      return true;
+    }
+
+    return false;
```

Comment:
If we don't change the error type, I would change this to

```
return String(error).includes('404')
```
since `includes` returns a bool.

If we do change the error type I would change this to
```
return (error as HttpError).status == 404
```

I am in favor of updating that type because the check as it stands could return true if `"404"` is anywhere in `statusText` as well